integration-cli/docker_api_containers_test.go

```diff
@@ -98,46 +96,7 @@ func (s *DockerSuite) TestContainerAPIGetJSONNoFieldsOmitted(c *check.C) {
 	}
 }
 
-type containerPs struct {
-	Names []string
-	Ports []types.Port
-}
-
-// regression test for non-empty fields from #13901
-func (s *DockerSuite) TestContainerAPIPsOmitFields(c *check.C) {
```

Comment:
See my other comments; looking at this again, I guess these were using `checker.NotNil` because the value itself was not important (but must be set), so perhaps we can change these to `value != ""` (e.g.)